src/main/java/core/basesyntax/handlers/AddFruitQuantity.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.handlers;
+
+import core.basesyntax.db.FruitStorage;
+
+public class AddFruitQuantity implements OperationHandler {

```

Comment:
1) Verbs are used for methods naming
2) Strange to see BalanceHandler, PurchaseHandler and AddFruitQuantity as equal implemetations of single interface
```suggestion
public class QuantityAdditionHandler implements OperationHandler {

```